port/unix/omrsysinfo.c

```diff
@@ -1718,7 +1727,12 @@ omrsysinfo_get_limit(struct OMRPortLibrary *portLibrary, uint32_t resourceID, ui
 	Trc_PRT_sysinfo_get_limit_Entered(resourceID);
 
 	if (OMRPORT_RESOURCE_ADDRESS_SPACE == resourceRequested) {
+#if !defined(OMRZTPF)
 		resource = RLIMIT_AS;
+#else /* !defined(OMRZTPF) */
+		resource = OMRPORT_LIMIT_UNKNOWN;
```

Comment:
Nice catch.
Also added the trace exit hook and actually updated the 'rc' return code instead of the resource variable which was intended to be passed onto getrlimit.